server.js

```diff
@@ -136,17 +137,68 @@ function server_setup(db) {
 				return;
 			} else {
 				console.log("got keys", cred);
-				titterPicture.drawComment(text, function(error, filename) {
-					titterSender.comment(cred, title + '\n' + message + ' Donate 0 WRG', filename, function done(err, res) {
-						if (err) {
-							response.status(400);
-							response.send(err);
-						} else {
-							response.send('Done');
-						}
-					});
-
-				});
+				request.files.forEach(function(e, i) {
```

Comment:
@balthazzar Callback hell, very hard to read code, i would suggest you to move some code to functions to flatten code.